country-info.csv

```diff
@@ -218,3 +219,5 @@ Seoul National University,asia
 Middlesex University,europe
 Newcastle University,australasia
 University of Surrey,europe
+Tilburg University,europe
```

Comment:
Please remove this and the next entry unless all CS faculty are added.

Reply:
added all cs faculty.